subworkflows/local/variant_calling/call_sv_germlinecnvcaller.nf

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env nextflow
```

Comment:
Is it the new standard to add a shebang for nextflow subworkflows? We should add it to the subworkflows as well then (another PR).

Reply:
Nope.. removed it.